di_website/templates/includes/cards/card-blog.html

```diff
@@ -22,28 +22,36 @@ <h3 class="card__title"><span>{{ blog.title }}</span></h3>
       </div>
 
       {% if blog.internal_author_page %}
-
         {% with blog.internal_author_page.specific as internal %}
           {% with page=internal img=internal.image name=internal.name position=internal.position %}
             {% include 'includes/partials/author.html' with author_prefix="Written by" %}
           {% endwith %}
         {% endwith %}
 
       {% elif blog.other_authors %}
+        {% if blog.count_other_authors > 1 %}
 
-        {% with other_author=blog.other_authors|first %}
-          {% if other_author.block_type == 'internal_author' %}
-            {% with other_author.value.specific as internal %}
-                {% with page=internal img=internal.image name=internal.name position=internal.position %}
-                    {% include 'includes/partials/author.html' with author_prefix="Written by" %}
-                {% endwith %}
-            {% endwith %}
-          {% else %}
-            {% with link=other_author.value.page img=other_author.value.photograph name=other_author.value.name position=other_author.value.title %}
-                {% include 'includes/partials/author.html' with author_prefix="Written by" %}
-            {% endwith %}
-          {% endif %}
-        {% endwith %}
+          {% with authors=blog.other_authors %}
+            {% include 'includes/partials/multi_authors.html' with author_count=blog.count_other_authors %}
+          {% endwith %}
+
+        {% else %}
+
+          {% with other_author=blog.other_authors|first %}
+            {% if other_author.block_type == 'internal_author' %}
+              {% with other_author.value.specific as internal %}
+                  {% with page=internal img=internal.image name=internal.name position=internal.position %}
+                      {% include 'includes/partials/author.html' with author_prefix="Written by" %}
+                  {% endwith %}
+              {% endwith %}
+            {% else %}
+              {% with link=other_author.value.page img=other_author.value.photograph name=other_author.value.name position=other_author.value.title %}
+                  {% include 'includes/partials/author.html' with author_prefix="Written by" %}
+              {% endwith %}
```

Comment:
I'm not seeing the markup for `Guest author`
![image](https://user-images.githubusercontent.com/5672438/85251972-69528880-b463-11ea-894f-5972351df223.png)

Look closely https://development-initiatives.surge.sh/page-templates/11-blog-listing

Reply:
Added `<span>` for guest author